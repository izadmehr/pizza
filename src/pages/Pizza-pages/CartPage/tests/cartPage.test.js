import React from "react";
import { shallow } from "enzyme";

import { CartPage } from "../index";

describe("CartPage Component", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<CartPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
