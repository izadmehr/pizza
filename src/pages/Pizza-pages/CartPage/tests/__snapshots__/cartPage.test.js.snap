// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartPage Component renders correctly 1`] = `
<section>
  <styled.h1>
    Let's Make Some Pizza!
  </styled.h1>
</section>
`;
